src/PKSim.UI/Views/ProteinExpression/ExpressionDataView.cs

```diff
@@ -698,23 +698,23 @@ private void configChart(string titleText)
          var chartBarView = chart.SeriesTemplate.View as SideBySideBarSeriesView;
          if (chartBarView != null) chartBarView.EqualBarWidth = true;
 
-         // Create argument descriptions for the summary function.
-         // Average
-         var argument1Description =
-            new SummaryFunctionArgumentDescription("Average", ScaleType.Numerical);
-         // Count
-         var argument2Description =
-            new SummaryFunctionArgumentDescription("Count", ScaleType.Numerical);
-
-         // Register the summary function in chart.
-         const string STR_CUSTOM_SUMMARY_FUNCTION_NAME = "WeightedAverage";
-         const string STR_CUSTOM_SUMMARY_FUNCTION_DISPLAY_NAME = "WeightedAverage";
-         chart.RegisterSummaryFunction(STR_CUSTOM_SUMMARY_FUNCTION_NAME, STR_CUSTOM_SUMMARY_FUNCTION_DISPLAY_NAME, 1,
-                                       new[] {argument1Description, argument2Description},
-                                       customSummary);
-
-         chart.SeriesTemplate.NumericSummaryOptions.SummaryFunction = 
-            $"{STR_CUSTOM_SUMMARY_FUNCTION_NAME}([{_fieldNormValue.FieldName + "_" + _fieldNormValue.SummaryType}],[{_fieldNormValue2.FieldName + "_" + _fieldNormValue2.SummaryType}])";
+         // // Create argument descriptions for the summary function.
```

Comment:
This is just required for now so that I can get some values. We can uncomment when the problem is solved with query